client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/get-thank-you-page-url.ts

```diff
@@ -277,16 +278,20 @@ export default function getThankYouPageUrl( {
 		return newBlogReceiptUrl;
 	}
 
-	const redirectUrlForPostCheckoutUpsell = receiptIdOrPlaceholder
-		? getRedirectUrlForPostCheckoutUpsell( {
-				receiptId: receiptIdOrPlaceholder,
-				cart,
-				siteSlug,
-				hideUpsell: Boolean( hideNudge ),
-				domains,
-				isDomainOnly,
-		  } )
-		: undefined;
+	// disable upsell for tailored signup users
+	const isTailoredSignup = isNewsletterOrLinkInBioFlow( signupFlowName );
```

Comment:
Could we add a test for this forked behavior [around here](https://github.com/Automattic/wp-calypso/blob/22408ac1261f90000e51ac7af681d95b68f04872/client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.ts#L1486) or wherever is appropriate?

Reply:
Also, I moved this function's location in https://github.com/Automattic/wp-calypso/pull/67123. Sorry for the inconvenience! 😅